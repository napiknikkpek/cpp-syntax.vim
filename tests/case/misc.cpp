auto& from = *xs[boost::hana::size_t<0 /*asdf*/>{}];
