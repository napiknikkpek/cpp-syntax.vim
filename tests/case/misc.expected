aaaa..xxxx....xx.ttttt..tttt..tttttt.c.mmmmmmmm.....
